### DISCUSSION TITLE
❌ Invalid environment variables

### DISCUSSION THREAD

**Discussion by MrWeiDaren** [reworkd/AgentGPT#507] [Q&A] [answered]
When I try to run"./setup.sh". It always comes back with an error:

`❌ Invalid environment variables:
 NEXT_PUBLIC_BACKEND_URL: Required

file:///Users/username/Documents/AutoGPT/AgentGPT/next/src/env/client.mjs:22
  throw new Error("Invalid environment variables");
        ^

Error: Invalid environment variables
    at file:///Users/username/Documents/AutoGPT/AgentGPT/next/src/env/client.mjs:22:9
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
`
Can someone please tell me how to fix this?

**Comment by asim-shrestha** [comment, marked as answer]
Hey - apologies, fixed by @SnoopyDevelops in https://github.com/reworkd/AgentGPT/pull/509
